pkg/virtctl/vm/vm_test.go

```diff
@@ -1,30 +1,60 @@
-package vm
+package vm_test
 
 import (
-	"testing"
+	"errors"
 
+	"github.com/golang/mock/gomock"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/tests"
 )
 
-var _ = Describe("VirtualMachine", func() {
+var _ = Describe("OfflineVirtualMachine", func() {
```

Comment:
change OfflineVirtualMachine -> VirtualMachine under each test

Reply:
oops... missed that somehow.. Thanks